readme.md

```diff
@@ -1,6 +1,6 @@
 <p align="center">
     <img src="https://github.com/rose-pine/rose-pine-theme/raw/main/assets/icon.png" width="80" />
-    <h2 align="center">Rosé Pine for GNOME Terminal</h2>
+    <h2 align="center">Rosé Pine Dawn for GNOME Terminal</h2>
```

Comment:
The dawn variant should be in addition, not instead of the base variant

Reply:
sorry i forgot about this! i fixed it and will make a new PR!